cmd/kubeconfig_generator/main.go

```
@@ -131,15 +132,15 @@ func main() {
 		klog.Fatalf("cannot save kubeconfig: %v", err)
 	}
 
-	klog.Infof("kubeconfig %q is saved", kubeconfigPath)
+	klog.Infof("kubeconfig %q is saved", kubeconfigPath.Path())
 
 	// wait for signal
 	sigterm := make(chan os.Signal, 1)
 	signal.Notify(sigterm, syscall.SIGINT, syscall.SIGTERM, syscall.SIGKILL)
```

golangci-lint (2.12.2)

[error] SA1016: syscall.SIGKILL cannot be trapped (did you mean syscall.SIGTERM?)

(staticcheck)